### DISCUSSION TITLE
JAX 101 - link to pytrees not working

### DISCUSSION THREAD

**Discussion by D3Reo** [jax-ml/jax#6061] [Q&A] [answered]
Hi, I was following the new jax 101 tutorial, the very first notebook (["JAX as accelerated NumPy"](https://colab.research.google.com/github/google/jax/blob/master/docs/jax-101/01-jax-basics.ipynb#scrollTo=yQAMTnZSqo-t)) briefly mentions pytrees, leaving them for another 101 while linking the reader to "https://colab.research.google.com/github/google/jax/blob/master/docs/jax-101/06-pytrees.ipynb". Unfortunately this link does not seem to be working, and in fact the sixth jax 101 is not on pytrees but on parallelism.
I know jax 101 is a wip, if this is something already known to you please ignore it.

I believe I need to tag @jakevdp for this one

Cheers! :)

**Comment by jakevdp** [comment, marked as answer]
Thanks - it's still a work in progress.

**Reply by D3Reo** [reply]
Yeah, I figured that would possibly be the reason. Thanks anyway for answering, I hope I'll have the change to read about pytrees soon! :)

**Reply by 8bitmp3** [reply]
I'm sure you've seen the PyTrees section in the docs but just in case here it is https://jax.readthedocs.io/en/latest/pytrees.html 👍
